pkg/bundle/operator.go

```diff
@@ -1 +1,231 @@
 package bundle
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/joelanford/ignore"
+	imgreference "github.com/openshift/library-go/pkg/image/reference"
+	"github.com/openshift/oc/pkg/cli/admin/catalog"
+	"github.com/openshift/oc/pkg/cli/image/imagesource"
+	imagemanifest "github.com/openshift/oc/pkg/cli/image/manifest"
+	imgmirror "github.com/openshift/oc/pkg/cli/image/mirror"
+	"github.com/operator-framework/operator-registry/pkg/action"
+	"github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/util/yaml"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+)
+
+func GetOperators(cfg *BundleSpec, rootDestDir string, dryRun bool) (err error) {
+
+	tmp, err := os.MkdirTemp(rootDestDir, "operators.*")
+	if err != nil {
+		return err
+	}
+	defer func() {
+		if err := os.RemoveAll(tmp); err != nil {
+			logrus.Error(err)
+		}
+	}()
+
+	for _, ctlg := range cfg.Operators {
+		// TODO(estroz): registry
+		catLogger := logrus.WithField("ctlg", ctlg.Catalog)
+		a := action.Diff{
```

Comment:
Running the diff library on "full" mode is temporary, just playing around with the mirroring setup. We can mirror full catalog images directly (without rendering anything to disk first).

Reply:
Fixed.